2021q2/multicast.adoc

```diff
@@ -5,7 +5,7 @@ Contact: Wojciech Macek <wma@semihalf.com>
 The Semihalf team has been working on reworking existing ip_mroute module. Previous implementation was over 20 years old and didn't use modern kernel features.
 
 Complete rework of locking mechanism was done to eliminate taking multiple locks for a single job. Some portions of code were modified to use lockless constructions, BW-meter API refactored to work in a single context and routing hot path was identified and made to work efficiently.
-All these fixes helped to acheive over 5x speed boost in multicast routing.
+All these fixes helped to achieve over 5x speed boost in multicast routing.
```

Comment:
I missed this earlier (sorry): s/helped to achieve/helped achieve/

Reply:
Sure, no pb - done. Thanks!